packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/create-serviceapp-grantscopes/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Create a service app and grant scopes
+---
+
+Create an OAuth service client app and register the public key with the service app using the dynamic client registration endpoint (`/oauth/v1/clients`). Then, grant the scopes that you want to allow for the service app.
+
+## Create a service app
+
+1. Use the following POST example to create your service app using the `/oauth2/v1/clients` endpoint and provide values for these parameters:
+
+    * `client_name` &mdash; Name of the service client app
+    * `grant_types` &mdash; `client_credentials`
+    * `token_endpoint_auth_method` &mdash; `private_key_jwt`
+    * `application_type` &mdash; `service`
+    * `jwks` &mdash; Add the JSON Web Key Set (JWKS) that you created in the <GuideLink link="../create-publicprivate-keypair">last step</GuideLink>.
+
+```JSON
+    curl -X POST \
+    -H 'Accept: application/json' \
+    -H "Authorization: SSWS ${api_token}" \
+    -H 'Content-Type: application/json' \
+    -d ' {
+        "client_name": "Service Client Name",
+        "response_types": [
+          "token"
+        ],
+        "grant_types": [
+          "client_credentials"
+       ],
+        "token_endpoint_auth_method": "private_key_jwt",
+        "application_type": "service",
+        "jwks": {
+              "keys": [
+                     {
+                       "kty": "RSA",
+                       "e": "AQAB",
+                       "use": "sig",
+                       "kid": "O4O",
+                       "alg": "RS256",
+                       "n": "u0VYW2-76A_lYg5NQihhcPJYYU9-NHbNaO6LFERWnOUbU7l3MJdmCailwSzjO76O-2GdLE-Hn2kx04jWCCPofnQ8xNmFScNo8UQ1dKVq0UkFK-sl-Z0Uu19GiZa2fxSWwg_1g2t-ZpNtKCI279xGBi_hTnupqciUonWe6CIvTv0FfX0LiMqQqjARxPS-6fdBZq8WN9qLGDwpjHK81CoYuzASOezVFYDDyXYzV0X3X_kFVt2sqL5DVN684bEbTsWl91vV-bGmswrlQ0UVUq6t78VdgMrj0RZBD-lFNJcY7CwyugpgLbnm4HEJmCOWJOdjVLj3hFxVVblNJQQ1Z15UXw"
+                       }
+                     ]
+           }
+    }' "https://${yourOktaDomain}/oauth2/v1/clients"
+```
+
+2. Make note of the `client_id` that is returned in the response. You need that to grant scopes to your service app and when you create and sign the JWT.
+
+### Grant allowed scopes
+
+When a request is sent to the Okta Org Authorization Server, it validates all of the requested scopes in the OAuth2 token request against the service client app's grants collection. The scope is granted if the scope exists in the service client app's grants collection.
```

Comment:
service client app -> service app 
Since in all previous paragraphs, we've used service app for consistency.